vite/public/edge-surf-game-v1/resources/js/parse_html_subset.js

```
@@ -0,0 +1,247 @@
+/**
+ * Parses a very small subset of HTML. This ensures that insecure HTML /
+ * javascript cannot be injected into WebUI.
+ * @param {string} s The string to parse.
+ * @param {!Array<string>=} opt_extraTags Optional extra allowed tags.
+ * @param {!Array<string>=} opt_extraAttrs
+ *     Optional extra allowed attributes (all tags are run through these).
+ * @throws {Error} In case of non supported markup.
+ * @return {!DocumentFragment} A document fragment containing the DOM tree.
+ */
+/* #export */ const parseHtmlSubset = (function() {
+  'use strict';
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)